matson.theme

```diff
@@ -72,3 +75,16 @@ function matson_preprocess_block(&$vars) {
 //     array_splice($suggestions, 2, 0, 'block__' . $bundle);
 //   }
 // }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
```

Comment:
I originally added that because D8 doesn't have a way of grabbing a unique template file per content type, the way D7 does.  This was the best solution I could find.

I ended up not needing it because I didn't create a template file for this content type.  However, this is a useful tool and I think there's a good chance you'll want it later.  It's right below a block template suggestion which is also commented out (I didn't add that one).

I'm happy to remove it, just wanted to double check.